### PR TITLE
$resaction need in 5.0.x too

### PR DESCRIPTION
**Pull request by defrance**

### CODE REVIEW

**Comment by hregis** [comment]
@defrance it's wrong !
if you use $resaction here, it can crush the $resaction of hook type "addreplace" !!

**Comment by defrance** [comment]
@hregis you have already this correction on development branch
you don't have see the else just before the addreplace
without this change, you don't have the $result on hook returned and testable

**Comment by hregis** [comment]
@defrance ok ok !!! :-)
c'est la faute de la chaleur !!

File: htdocs/core/class/hookmanager.class.php [hunk 1]
```diff
@@ -233,7 +233,7 @@ function executeHooks($method, $parameters=false, &$object='', &$action='')
                     	if (is_array($parameters) && ! empty($parameters['special_code']) && $parameters['special_code'] > 3 && $parameters['special_code'] != $actionclassinstance->module_number) continue;
 
                     	//dol_syslog("Call method ".$method." of class ".get_class($actionclassinstance).", module=".$module.", hooktype=".$hooktype, LOG_DEBUG);
-                    	$result = $actionclassinstance->$method($parameters, $object, $action, $this); // $object and $action can be changed by method ($object->id during creation for example or $action to go back to other action for example)
+                    	$resaction = $actionclassinstance->$method($parameters, $object, $action, $this); // $object and $action can be changed by method ($object->id during creation for example or $action to go back to other action for example)
```
Comment (eldy) [inline htdocs/core/class/hookmanager.class.php:236]: If you change result into resaction action, you must also change result in following lines

**Pull request merged**